cmd/minikube/cmd/status.go

```diff
@@ -170,6 +170,12 @@ type BaseState struct {
 	StepDetail string `json:",omitempty"`
 }
 
+// ScheduledStopStatus holds string representation of scheduledStopDuration
+type ScheduledStopStatus struct {
```

Comment:
This struct should be part of the top level `Status` struct (Line 131)


```go

type Status struct {
ScheduledStop ScheduledStopStatus
}
```

Reply:
@priyawadhwa I was thinking since the `ScheduledStopStatus` struct has the only field, we can keep that in `status` as `TimeToStop string`? Do we need to also need to show `type: ScheduledDuration` in status ?

Reply:
That's a good question -- let's keep it as a struct in case we ever want to add more information to it.

I think we can start with just `TimeToStop` and we can think about `ScheduledDuration` later if we decide we need it.